link/tracing_test.go

```diff
@@ -82,17 +82,29 @@ func TestTracing(t *testing.T) {
 		},
 	}
 
+	test := func(
```

Comment:
This is really hard to read, sorry. Could an entry with the default attach type be added to `tests` instead?